docs/development/releasing.rst

```diff
@@ -175,26 +171,28 @@ packages that use the full bugfix/maintenance branch approach.)
 
 #. Push up all these changes to the `astropy core repository`_::
 
-      $ git push --tags upstream v<version>
+      $ git push --tags upstream v<version branch>.x
```

Comment:
With this I've managed to mess up the stable tag yesterday (after backporting to 1.0.x). Locally I have a stable tag on 5f68a382adcf2b560cb4579b7170052f079041d2, together with 0.2.1. When using this command it may have been pushed up, too as e.g. travis has picked it up here: https://travis-ci.org/astropy/astropy/branches

I'm a bit confused on where it went wrong,and whether we had a ``stable`` tag before (as well as the branch), or that was deleted long time ago?